VKappProject/VKappProject/Constants.swift

```diff
@@ -55,6 +55,9 @@ enum Constants {
         static let newsViewControllerIdentifier = "news"
         static let segueFriendImagesIdentifier = "friendImages"
         static let storyBoard = "Main"
```

Comment:
```suggestion
        static let storyBoard = "Main"
```
какой сториборд конкретно их может быть и 3 и 10